src/NinjaTools.FluentMockServer.API/Types/ChainElement.cs

```diff
@@ -0,0 +1,134 @@
+namespace NinjaTools.FluentMockServer.API.Types
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty namespace.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)